### DISCUSSION TITLE
Music sensor - add more properties

### DISCUSSION THREAD

**Discussion by juan518munoz** [zepp-health/zeppos-forum#78] [Developer Feedback]
As visible on the embedded music player on ZeppOS, the song duration and current progress is accessible to the watch, but not on the current sensor API. It would be handy if `length` and `playbackPosition` are added.

https://docs.zepp.com/docs/reference/device-app-api/hmSensor/sensorId/MUSIC/#creating-sensors

**Comment by htoooth** [comment]
hi @juan518munoz: 
Thank you for your feedback. We will add this feature to our version plan.
